frontend/src/components/archive/ArchiveListItem.tsx

```
@@ -0,0 +1,165 @@
+                aria-label={`Delete archive ${name}`}
+                disabled={isDeleting}
+                aria-busy={isDeleting}
+              >
+                {isDeleting ? 'Deleting...' : 'Delete'}
+              </button>
+            )}
+          </div>
+        </div>
+
+        {/* T115: Enhanced ARIA labels for statistics */}
+        <div className="flex gap-4 text-sm" role="list" aria-label="Archive statistics">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)